e2e/definitions/backup_repository/README.md

```diff
@@ -0,0 +1,122 @@
+# A restic Backup
+
+This folder contains a complete restic backup.
+`restic snapshot` would show it like this:
+
+```bash
+$ restic snapshots
+ID        Time                 Host              Tags        Paths
+------------------------------------------------------------------------------
+fcf582cd  2021-03-09 15:48:32  k8up-e2e-subject              /data/subject-pvc
+------------------------------------------------------------------------------
+1 snapshots
+```
+
+## Re-create the backup
+
+To create the backup, I ran the test `test-01-backup.bats`.
+But I wanted to have the string `MagicString` as content of the file `/data/expectation.txt` in the `subject` container,
+which is actually the `/expectation.txt` file in the `subject-pv`.
+So I changed the deployment of the subject:
+
+```patch
+--- a/e2e/definitions/subject/deployment.yaml
++++ b/e2e/definitions/subject/deployment.yaml
```

Comment:
Is this some sort of blog, manual or your personal notes? I'm not quite sure what the intent with this README is :)

Reply:
There are two intents:
- It's a note what this folder's purpose is, since it contains "weird" files.
- It's the instruction how to re-create that backup should it be needed.

Reply:
Would it be better to put it somewhere in the asciidoc?

Reply:
I think so. I remember some discussion that we want to create a developer guide/section anyway.

Could we at least have a script that does the recreation for us?